db/views/rejection_percentage_by_prison_and_calendar_weeks_v02.sql

```diff
@@ -0,0 +1,65 @@
+SELECT rejected.prison_name,
```

Comment:
I have now.  Unfortunately, adapting the existing queries simply causes the same bug to reappear.  I made some progress with them, but ultimately wasn't getting the desired results in a timely fashion.  This works for now, so I will leave it as is and revisit later.